xsd_glue/domain_getinfo_plugin_v1/domain_getinfo_stubs_v1.c

```diff
@@ -43,7 +43,7 @@ static struct custom_operations xsd_glue_xenctrl_ops = {
         .compare_ext = custom_compare_ext_default,      /* Can't compare     */
 };
 
-static void Noreturn
```

Comment:
docs in `runtime/caml/misc.h` says that instead of `Noreturn` we should prefer:
`CAMLnoreturn_start foo CAMLnoreturn_end;`

Reply:
OK, I've updated the patch as this seems to work. Odd that none of these macros are in the OCaml docs (at least I couldn't find them).